mantis-client/src/main/java/io/mantisrx/client/SinkConnectionsStatus.java

```diff
@@ -16,27 +16,11 @@
 
 package io.mantisrx.client;
 
-public class SinkConnectionsStatus {
-
-    private final long numConnected;
-    private final long total;
-    private final long recevingDataFrom;
-
-    public SinkConnectionsStatus(long recevingDataFrom, long numConnected, long total) {
-        this.recevingDataFrom = recevingDataFrom;
-        this.numConnected = numConnected;
-        this.total = total;
-    }
+import lombok.Value;
 
-    public long getRecevingDataFrom() {
-        return recevingDataFrom;
-    }
-
-    public long getNumConnected() {
-        return numConnected;
-    }
-
-    public long getTotal() {
-        return total;
-    }
+@Value
```

Comment:
nit: this is a different change than the other changes in this diff. Let's add lombok annotations separately. The reason I'm slightly worried about lombok changes is that we do not know if these DS are being serialized by jackson somewhere. If we are planning on using jackson with these DS, then the lombok annotations will have to change or the lombok config has to change. It might be better to do these changes separately.